src/components/CorporateCampaign/CampaignProgressBar.vue

```diff
@@ -47,6 +47,32 @@
 			</kv-ui-button>
 		</div>
 	</div>
+	<div
+		v-else
+		class="
+			tw-rounded
+			tw-bg-brand-50
+			tw-w-full
+			tw-py-2
+			tw-px-1
+			tw-mb-2
+			tw-flex-col
+			tw-flex
+			tw-justify-start
+			lg:tw-justify-between
+			lg:tw-align-center
+			"
+	>
+		<div class="lg:tw-mr-1 lg:tw-ml-0 tw-ml-1 lg:tw-pt-1">
+			<kv-ui-button
+				v-show="basketLoans.length > 0"
+				@click="$emit('show-basket')"
+				class="tw-w-full"
+			>
+				Checkout now
+			</kv-ui-button>
+		</div>
+	</div>
```

Comment:
Does there need to be a 3rd UI fallback if the two cases in this template are not valid? So when it's not a matching campaign and there are no items in the basket?

Reply:
@dyersituations Yes I'm still unsure what the fallback should be and have improvised with just a empty progress bar section- I'll likely reach with Sara about it!